packages/as-transformer/src/transformers/file2ByteArray.ts

```diff
@@ -9,14 +9,20 @@ import { SimpleParser } from 'visitor-as';
 import * as fs from 'fs';
 
 /**
- * File2ByteArray transformer
+ * The File2ByteArray transformer object allows to transform writes a function call which aims to transform
```

Comment:
That's very unclear, you may not want to precise what a transformer is again here.
Just talk about the goal of this specific transformer
(PS: It doesn't change an AST into a byte array, it creates the following **expression**: 'transforms a file into a byte array'  and stores it into the AST as a **new AST Node** replacing the old **call expression** node)

Reply:
@gregLibert tell me if i'm right

Reply:
This tranformer is updating the AST by:
- replacing the node containing a call to the fileToByteArray function having a file as argument
- by a node containing an array of bytes corresponding to the content of the file given as argument.
Does it make sens :thinking: